content/version/1/4/code-of-conduct.md

```diff
@@ -28,34 +28,35 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
-  advances
+  advances after being advised of its unwelcomeness
 * Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
+* Repeated public or private harassment
 * Publishing others' private information, such as a physical or electronic
   address, without explicit permission
+* Attacking or villainizing project members over political or personal
+  beliefs and statements which are unrelated to the project
 * Other conduct which could reasonably be considered inappropriate in a
   professional setting
 
 ## Our Responsibilities
 
 Project maintainers are responsible for clarifying the standards of acceptable
-behavior and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
+behavior and are expected to take appropriate, fair, and just corrective action
+in response to any instances of unacceptable behavior.
 
 Project maintainers have the right and responsibility to remove, edit, or
 reject comments, commits, code, wiki edits, issues, and other contributions
 that are not aligned to this Code of Conduct, or to ban temporarily or
-permanently any contributor for other behaviors that they deem inappropriate,
-threatening, offensive, or harmful.
+permanently any contributor for other project-related behaviors that they 
+deem inappropriate, threatening, offensive, or harmful.
 
 ## Scope
 
 This Code of Conduct applies both within project spaces and in public spaces
 when an individual is representing the project or its community. Examples of
 representing a project or community include using an official project e-mail
 address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
```

Comment:
This restricts the freedom of project maintainers to define representation to be as restrictive or inclusive as they need it to be.

Reply:
No, it merely protects members from being attacked for things they do or say while not representing the project.  Otherwise someone could dredge up dirt on someone, say "this counts as representation", and cause all kinds of unwelcome drama over something not actually related to the project.

If a particular project has other types of representation, they can add those specifically to their CoC.  However, as it stands currently it's too broad.

Reply:
You removed the clause that says, "If a particular project has other types of representation, they can add those specifically to their CoC."

Reply:
That's what the CC-BY license is for, and why the attribution in the footer is "This Code of Conduct is *adapted* from the Contributor Covenant, version 1.4", not "This Code of Conduct *is* the Contributor Covenant, version 1.4".